pkg/api/dynamicaccess.go

```diff
@@ -113,6 +113,8 @@ func (s *Service) actDecryptionHandler() func(h http.Handler) http.Handler {
 			ls := loadsave.NewReadonly(s.storer.Download(cache))
 			reference, err := s.dac.DownloadHandler(ctx, ls, paths.Address, headers.Publisher, *headers.HistoryAddress, timestamp)
 			if err != nil {
+				logger.Debug("act failed to decrypt reference", "error", err)
+				logger.Error(nil, "act failed to decrypt reference")
```

Comment:
Why is it logging `nil` to error level and log the error to debug?

Reply:
this pattern is used in other enpoints (I agree it is strange)